tools/docker/bazel.dockerfile

```diff
@@ -5,7 +5,7 @@ ARG TARGETARCH
 
 COPY --chown=0:0 bazel/install-deps.sh /
 
-RUN apt-get update && apt install -y wget python3 python3-venv curl pigz patchelf devscripts debhelper rpm zip
+RUN apt-get update && apt install -y wget python3 python3-venv curl pigz patchelf devscripts debhelper rpm zip default-jre
```

Comment:
We could also manage the JRE in bazel.

Reply:
Yeah I was wondering, might have a look later.